libs/blocks/quiz/quiz.js

```diff
@@ -294,41 +294,42 @@ const App = ({
   };
 
   return html`<div class="quiz-container">
-                  <${StepIndicator} 
+                  ${selectedQuestion.questions && html`<${StepIndicator}
                     currentStep=${currentStep} 
                     totalSteps=${totalSteps} 
                     prevStepIndicator=${prevStepIndicator}
                     top="${true}" />
+                  `}
 
-                  <div class="quiz-background">
+                  ${selectedQuestion.questions && html`<div class="quiz-background">
```

Comment:
The background element should be contingent on whether one has actually been specified in the json. Otherwise, we just create a picture element with an empty src, which renders as a broken image. 

`${selectedQuestion.questions && getStringValue('background') !== '' && html`<div class="quiz-background">`

The two instances of getStringValue also need to be made more robust. Respectively, I think they need to look like the following:

`const getStringValue = (propName) => {
  if (!selectedQuestion?.questions) return '';
  const question = stringQList[selectedQuestion.questions];
  return question?.[propName] || '';
};`

`const getStringValue = (propName) => {
  if (!selectedQuestion?.questions) return '';
  const question = stringQList[selectedQuestion.questions];
  return question?.[propName] || '';
};`

Otherwise, the methods can happily be called when selectedQuestion might be valid, but selectedQuestion.questions isn't, and they error in their current forms.

Reply:
Done. Updated. Thanks for the feedback.